lib/jsonnet/evaluator.rb

```diff
@@ -0,0 +1,46 @@
+module Jsonnet
+  class Evaluator
```

Comment:
* What is this class is for?
   I don't have a clear understanding of use cases of instances of this class. So I suspect if this class is just redundant.
* Also `Evaluator.from_snippet` sounds as if it returns an instance of `Evaluator`.  I guess this confusion is also coming from the same issue of redundancy.
* Feel free to just add this kind of higher-level convenient methods to `VM` class if necessary.